plugins/modules/s3_bucket.py

```diff
@@ -84,6 +84,12 @@
     description: KMS master key ID to use for the default encryption. This parameter is allowed if I(encryption) is C(aws:kms). If
                  not specified then it will default to the AWS provided KMS key.
     type: str
+  encryption_bucket_key:
+    description: Bucket Key for SSE-KMS. This parameter is allowed only if encryption is aws:kms otherwise it will throw error.
```

Comment:
```suggestion
    description:
      - Enable S3 Bucket Key for SSE-KMS on new objects.
      - See the AWS documentation for more information
        U(https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucket-key.html).
      - Bucket Key encryption is only supported if I(encryption=aws:kms).
```

Reply:
Not for other reviewers - "S3 Bucket Keys for SSE-KMS" feels a little clumsy, however that's the name of the feature/product.